primefaces/src/main/java/org/primefaces/model/feedreader/FeedItem.java

```diff
@@ -0,0 +1,260 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2009-2024 PrimeTek Informatics
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.primefaces.model.feedreader;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Class representing a RSS item. A channel may contain any number of items. An item may represent a "story" -- much
+ * like a story in a newspaper or magazine; if so its description is a synopsis of the story, and the link points
+ * to the full story.
+ */
+public class FeedItem implements Serializable {
```

Comment:
I'm thinking if instead of listing every attributes, to have a ctor with Item and all this class does is to delegate to the item. There is one hiccup, class Item from the lib doesn't seem to be serializable, maybe we could request to make it so

Reply:
I thought of that too but chose this way ONLY to isolate the only place `com.apptasticsoftware.rssreader` would be is in the RSSutils and this FeedItem would just be a Serializable POJO.